lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -121,16 +121,17 @@ def test_subsection_update_triggers_signal(self, mock_subsection_signal):
         self.assertTrue(mock_subsection_signal.called)
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 1),
-        (ModuleStoreEnum.Type.split, 0),
+        (ModuleStoreEnum.Type.mongo, 1, 23),
+        (ModuleStoreEnum.Type.split, 3, 22),
     )
     @ddt.unpack
-    def test_subsection_grade_updated(self, default_store, added_queries):
+    def test_subsection_grade_updated(self, default_store, num_mongo_calls, num_sql_calls):
         with self.store.default_store(default_store):
             self.set_up_course()
             self.assertTrue(PersistentGradesEnabledFlag.feature_enabled(self.course.id))
-            with check_mongo_calls(2) and self.assertNumQueries(22 + added_queries):
```

Comment:
**Note to @sanfordstudent:** When the query context manager is used in a conjunction, only the latter query count is checked.  Not the former.  So all the `check_mongo_calls` in this file were not being called.  :(